### ISSUE TITLE
Fling not working

### ISSUE THREAD

**Issue by soulpoison** [Jawnnypoo/PhysicsLayout#9]
I have app:fling="true" and app:physics="true" in my PhysicsFrameLayout, but I cant't fling the ImageVew inside.

Is there something extra I need to do?

**Comment by Jawnnypoo** [comment]
Hey there, do you mind posting the full XML? It could be that your physics layout is not large enough to allow for flinging so you could try setting it's width and height manually to something like 200dp. Another possibility is that you are setting a onTouch listener on the physics layout which is overriding the one that is set when you set flinging to true. Maybe try running the sample app on the Play Store and see if you have the problem on your device and if it is device specific.

**Comment by soulpoison** [comment]
The sample app works perfectly. Here is the full xml file.

```
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:fab="http://schemas.android.com/apk/res-auto"
    xmlns:app="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.andexert.library.RippleView
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <com.jawnnypoo.physicslayout.PhysicsFrameLayout
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:id="@+id/physics_layout"
            android:background="@color/colorBlackTransparent"
            app:bounds="true"
            app:gravityX="0"
            app:gravityY="0"
            app:fling="true">

            <de.hdodenhof.circleimageview.CircleImageView
                android:layout_width="100dp"
                android:layout_height="100dp"
                android:src="@drawable/joxer"
                app:civ_border_width="2dp"
                app:civ_border_color="@android:color/transparent"/>

            <de.hdodenhof.circleimageview.CircleImageView
                android:layout_width="100dp"
                android:layout_height="100dp"
                android:src="@drawable/joxer"
                app:civ_border_width="2dp"
                app:civ_border_color="@android:color/transparent"/>

            <de.hdodenhof.circleimageview.CircleImageView
                android:layout_width="100dp"
                android:layout_height="100dp"
                android:src="@drawable/joxer"
                app:civ_border_width="2dp"
                app:civ_border_color="@android:color/transparent"/>

        </com.jawnnypoo.physicslayout.PhysicsFrameLayout>
    </com.andexert.library.RippleView>

    <com.github.clans.fab.FloatingActionMenu
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_alignParentBottom="true"
        android:layout_alignParentLeft="true"
        android:layout_alignParentStart="true"
        android:layout_marginRight="10dp"
        android:layout_marginBottom="10dp"
        android:layout_marginLeft="10dp"
        fab:menu_colorNormal="@android:color/transparent"
        fab:menu_colorPressed="@color/colorBlackTransparent"
        fab:menu_openDirection="up"
        fab:menu_labels_position="right"
        fab:menu_backgroundColor="@android:color/transparent"
        fab:menu_labels_colorNormal="@android:color/transparent"
        fab:menu_labels_colorPressed="@color/colorBlackTransparentTwo"
        fab:menu_labels_showShadow="false"
        fab:menu_showShadow="false"
        >

        <com.github.clans.fab.FloatingActionButton
            android:id="@+id/menu_item_create"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:src="@drawable/create_24"
            fab:fab_colorNormal="@color/colorBlackTransparentTwo"
            fab:fab_colorPressed="@color/colorBlack"
            fab:fab_size="normal"
            fab:fab_label="Create"/>

        <com.github.clans.fab.FloatingActionButton
            android:id="@+id/menu_item_search"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:src="@drawable/search_24"
            fab:fab_colorNormal="@color/colorBlackTransparentTwo"
            fab:fab_colorPressed="@color/colorBlack"
            fab:fab_size="normal"
            fab:fab_label="Search" />

        <com.github.clans.fab.FloatingActionButton
            android:id="@+id/menu_item_multiselect"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:src="@drawable/multiselect_24"
            fab:fab_colorNormal="@color/colorBlackTransparentTwo"
            fab:fab_colorPressed="@color/colorBlack"
            fab:fab_size="normal"
            fab:fab_label="Multiselect" />

        <com.github.clans.fab.FloatingActionButton
            android:id="@+id/menu_item_settings"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:src="@drawable/settings_24"
            fab:fab_colorNormal="@color/colorBlackTransparentTwo"
            fab:fab_colorPressed="@color/colorBlack"
            fab:fab_size="normal"
            fab:fab_label="Settings"/>

    </com.github.clans.fab.FloatingActionMenu>

</RelativeLayout>
```

**Comment by soulpoison** [comment]
Lol.... for some reason it magically works. I was tinkering with things and now my layout is really weird, but it works. Thank you for the quick support. This library is awesome! :+1:

**Comment by soulpoison** [comment]
Is there any reason why the above xml won't work? It doesn't just lose fling, but also keeps the default gravity and other attributes. The good thing is: it keeps the styling I make to the FABs.

What I did to "fix" this caused my FAB Menu and FABs to use their default styling even though I kept their styling in the xml. I'm not really sure why this is happening.

I'd like to have both the physicslayout and FAB menu, but I can't figure out the problem.

Whenever I make the PhysicsLayout a root view, I get an error inflating the xml layout. This is the root layout inside a fragment... I don't know if that has something to do with it.
